### ISSUE TITLE
Request: Revert to previous version of JPN font

### ISSUE THREAD

**Issue by telig** [hrydgard/ppsspp#6637]
Hi,
the dialog font for the NGE2: Tsukurareshi Sekai is unreadable. I have bisected the problem to the following commit:

0436b751a70c4a6b374f3dca777f13e0d42f8855 is the first bad commit
commit 0436b751a70c4a6b374f3dca777f13e0d42f8855
Author: Henrik Rydgard
Date:   Sun Aug 25 01:07:51 2013 +0200
Let's try W-MS' latest converted JPN font. If it's good we'll keep it, if not I'll just revert.

When I reverted the font to the previous version, it displays OK.

Example comparison from the menu (all dialogs are like that):
![comparison](https://cloud.githubusercontent.com/assets/8277719/3712372/8f1fbd02-150f-11e4-9840-b4cc0bc41084.png)

**Comment by unknownbrackets** [comment]
Is this still happening in the latest version?  I think we fixed some bugs in sceFont since this was reported, even if we have not reverted the font.

Just undoing progress isn't always the best way to make an emulator better.  If all we did was undo things whenever something wasn't prefect, we'd end up undoing everything and back to a single line of code, `printf("I will someday be a PSP emulator.\n");`.  Or if we undid the undos, we would just spend all our time doing that.  Forever.

-[Unknown]
